src/Observer.js

```diff
@@ -16,13 +26,36 @@ export default class {
         this.WebSocket.sendObj = (obj) => this.WebSocket.send(JSON.stringify(obj))
       }
     }
+
+    return this.WebSocket
+  }
+
+  reconnect () {
+    if (this.reconnectionCount > 0) {
+      this.reconnectionCount--
+      clearTimeout(this.reconnectTimeoutId)
+
+      this.reconnectTimeoutId = setTimeout(() => {
+        if (this.store) { this.passToStore('SOCKET_RECONNECT', this.reconnectionCount) }
```

Comment:
Please, add these mutations to readme.md:
SOCKET_RECONNECT & SOCKET_RECONNECT_ERROR